### ISSUE TITLE
Attempting to dump certificate to PEM string

### ISSUE THREAD

**Issue by dmwesterhoff** [wbond/asn1crypto#78]
Hi, maybe I am utilizing the API incorrectly but after creating a certificate with certbuilder I am trying to dump that certificate as a PEM encoded string.

`>>> der_bytes = root_ca_certificate.dump()`
`>>> pem_bytes = pem.armor('CERTIFICATE', der_bytes)`

And I'm receiving:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/asn1crypto/pem.py", line 82, in armor
    type_name(type_name)
TypeError: 'str' object is not callable
```

Any advice on how to proceed?

**Comment by dmwesterhoff** [comment]
I have solved the issue after looking through the source, had to make a slight change to the example code in the pem encode/decode docs:

`>>> pem_bytes = pem.armor(u'CERTIFICATE', der_bytes)`

Which ensured the 'CERTIFICATE' value is unicode

**Comment by wbond** [comment]
The bug you found (in regards to type checking) has been fixed in b7256ed36f786a79ce93c72a5c6bcbe2826c8832, and will be part of the next release.

Sorry for the confusion/error.
